### ISSUE TITLE
Invalid appdata metainfo file

### ISSUE THREAD

**Issue by Vogtinator** [lbellonda/qxmledit#63]
The appdata metainfo xml file is invalid:

https://github.com/lbellonda/qxmledit/blob/27f7bb6154035e8dc3e63f2c2fb4fcbb94aeafa3/install_scripts/environment/desktop/QXmlEdit.appdata.xml#L5

This was causing crashes in AppStream and narrowed down to this component extending itself:
https://github.com/ximion/appstream/issues/243

AFAICT just removing the `<extends>QXmlEdit.desktop</extends>` should fix this.

**Comment by lbellonda** [comment]
You are right. The current specifications do not allow the "extends" tag and some other tag needs an update. I file this issue for the next release due in a couple of month at maximum. Do you are able to apply a patch in the meantime?

**Comment by Vogtinator** [comment]
For now we applied a patch to AppStream to not crash, so for now it just rejects QXmlEdit (as it should).

**Comment by lbellonda** [comment]
An updated appInfo has been pushed to "194-invalid_metadata" branch. May you test it?
Please note that it references the .desktop file in "launchable" tag.

Thank you.

**Comment by Vogtinator** [comment]
Unfortunately I do not know how to test the .xml file, but

```
> appstreamcli validate QXmlEdit.appdata.xml 
Validation was successful: pedantic: 1
```

so it seems good.

**Comment by lbellonda** [comment]
Published a fix in 0.9.14.
